dds_cli/project_status.py

```diff
@@ -66,10 +69,28 @@ def get_status(self, show_history):
         except simplejson.JSONDecodeError as err:
             raise exceptions.APIError(f"Could not decode JSON response: {err}")
         else:
-            LOG.info(f"Current status of {self.project}: {resp_json.get('current_status')}")
+            current_status = resp_json.get("current_status")
+            current_deadline = resp_json.get("current_deadline")
+            status_out = f"Current status of {self.project}: {current_status}"
+            deadline_out = ""
+            if current_deadline:
+                date = pytz.timezone("UTC").localize(
+                    datetime.datetime.strptime(current_deadline, "%a, %d %b %Y %H:%M:%S GMT")
+                )
```

Comment:
Since this generated an error due to GMT before, maybe we should add a try/except block to catch that error and in that case just print the time or something?

Reply:
Just so we avoid a long exception that the user doesn't understand